Trappist/src/Promact.Trappist.Web/Views/Shared/_Layout.cshtml

```diff
@@ -48,5 +48,6 @@
         </div>
     </footer>
     <script src="https://cdn.ckeditor.com/4.6.2/standard/ckeditor.js"></script>
+    <script src="https://cdn.ckeditor.com/4.5.11/full-all/plugins/divarea/plugin.js"></script>
```

Comment:
If cdn is down the ckeditor is supposed to work as normal. It will only throw an warning when navigating back. This won't be an issue for this fix.